Makefile

```diff
@@ -456,6 +456,34 @@ dev-azure-nuke: envsubst azure-nuke ## Warning: Destructive! Nuke all Azure reso
 	$(AZURENUKE) run --config config/dev/azure-cloud-nuke.yaml --force --no-dry-run
 	@rm config/dev/azure-cloud-nuke.yaml
 
+
+KUBEVIRT_VERSION = $(shell curl -s https://storage.googleapis.com/kubevirt-prow/release/kubevirt/kubevirt/stable.txt)
+CDI_VERSION = $(shell basename $$(curl -s -w '%{redirect_url}' -o /dev/null https://github.com/kubevirt/containerized-data-importer/releases/latest))
```

Comment:
can we calculate those only within the `kubevirt` recipe? 

```makefile
.PHONY: kubevirt
kubevirt: CDI_VERSION = ...
kubevirt: KUBEVIRT_VERSION = ...
kubevirt
  # rest of the target
```

it'd be easier to refactor the `Makefile` someday if we take care of different variables decreasing their "scope"